airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -61,6 +61,9 @@
 
 
 class SourceGithub(AbstractSource):
+
+    continue_sync_on_stream_failure = False
```

Comment:
It's `False` by default [here](https://github.com/airbytehq/airbyte/blob/78a60475a4feb8c7263b6b5481a7ee96d98ee2bc/airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py#L285-L293) , was there a need to add this flag explicitly for visibility?

Reply:
my bad, thanks